src/main/java/driver/Finnhub.java

```diff
@@ -30,7 +30,7 @@ public Finnhub() {
         try {
             inputStream = new FileInputStream(f);
         } catch (FileNotFoundException e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException("Please create file.txt in the root directory with your Finnhub API key");
```

Comment:
Why throw a RuntimeException? I thought a new APIKeyNotFoundException was supposed to be thrown?